fastlane/lib/fastlane/actions/install_xcode_plugin.rb

```diff
@@ -4,6 +4,12 @@ class InstallXcodePluginAction < Action
       def self.run(params)
         require 'fileutils'
 
+        if params.fetch(:github, ask: false)
```

Comment:
Instead just define this option as `optional: true`, then you can just use `params[:github]`